srcs/layouts/latn_bone.xml

```diff
@@ -1,72 +1,131 @@
-<?xml	version="1.0"	encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <!-- https://neo-layout.org/Layouts/bone/ -->
-<keyboard name="Bone" bottom_row="false" script="latin">
-	<!-- first row + characters from number row:
+<keyboard name="Bone" script="latin">
+	<modmap>
+		<!-- shift on number row:
+			1234567890 -> °§ℓ»«$€„“”
+		-->
+		<shift a="1" b="°"/>
+		<shift a="2" b="§"/>
+		<shift a="3" b="ℓ"/>
+		<shift a="4" b="»"/>
+		<shift a="5" b="«"/>
```

Comment:
These are already `Fn+<` and `Fn+>`

Reply:
yeah the fn mapping conflicts with neo layouts, 
i was considering removing it or completely mod-remapping it for some other purpose

Reply:
Présumably, these symbols are on an other layer in the neo layout too ? I'm fine with remapping the entire Fn layer but the main layout seems crowded.